homework21/Product.java

```diff
@@ -0,0 +1,55 @@
+import java.time.LocalDateTime;
+
+public class Product {
+    private String typ;
+    private double price;
+    private boolean discount;
```

Comment:
The naming of the `boolean` variables is a bit different. I would suggest giving it a name `hasDiscount`.

Reply:
+++